### PR TITLE
Change the java 8 specification

### PR DESCRIPTION
**Pull request by leonard-IMBERT**
Change this to make it more global

### CODE REVIEW

File: runtimes/java/play-framework-2.md [hunk 1]
```diff
@@ -19,8 +19,7 @@ Refer to the page [Deploy an application on Clever Cloud](/clever-cloud-overview
 ## Necessary information
 
 * The application must be located at the **root** of the git repository.
-* Play 2.4 need **Java 8** to work. You need to [select Java 8](https://www.clever-cloud.com/doc/java/select-java-version/)
-for your application.
+* Your application may need **Java 8** to work (for example Play 2.4 need it). You can do it [this way](https://www.clever-cloud.com/doc/java/select-java-version/).
```
Comment (divarvel) [inline runtimes/java/play-framework-2.md]: needs* it.

Please read [select the java version](https://www.clever-cloud.com/doc/java/select-java-version/) for more information.

File: runtimes/scala/play-framework-2.md [hunk 2]
```diff
@@ -20,8 +20,7 @@ Refer to the page [Deploy an application on Clever Cloud](/clever-cloud-overview
 ## Necessary information
 
 * The application must be located at the **root** of the git repository.
-* Play 2.4 need **Java 8** to work. You need to [select Java 8](https://www.clever-cloud.com/doc/java/select-java-version/)
-for your application.
+* Your application may need **Java 8** to work (for example Play 2.4 need it). You can do it [this way](https://www.clever-cloud.com/doc/java/select-java-version/).
```
Comment (divarvel) [inline runtimes/scala/play-framework-2.md]: Starting from play 2.4, your application needs java 8 to run, please read [select the java version](https://www.clever-cloud.com/doc/java/select-java-version/) for more information.

**Comment by divarvel** [comment]
Please add it to the FAQ as well (with the exact error message copied for easier searching)

**Comment by leonard-IMBERT** [comment]
s/Vhange/Change

File: commons/get-help/faq.md [hunk 3]
```diff
@@ -121,3 +121,9 @@ can commit them in your application's Clever Cloud repository and then add the
 `clevercloud/ssh.json`.
 
 The ssh.json file is documented in the [common configuration page](/clever-cloud-overview/common-application-configuration/#private-ssh-key)
+
+## I got an `Unsupported major.minor version` error. How can I fix it ?
```
Comment (divarvel) [inline commons/get-help/faq.md]: I got a `Unsupported major.minor version` error on a java application. How can I fix it?

File: commons/get-help/faq.md [hunk 4]
```diff
@@ -121,3 +121,9 @@ can commit them in your application's Clever Cloud repository and then add the
...
+
+If you get this error with Java or Scala application, it's mean that you use a different Java version
+than us. We use by default Java 7 but you can change it by readind the [select java version](/doc/java/select-java-version/)
+part of the documentation.
```
Comment (divarvel) [inline commons/get-help/faq.md]: If you encounter this error on a java (or any JVM language) application, it means that your application requires a specific version of java. By default, java 7 is used, but you can change it. Please have a look at [select java version](/doc/java/select-java-version/) for more information.

File: runtimes/java/play-framework-2.md [hunk 5]
```diff
@@ -19,8 +19,7 @@ Refer to the page [Deploy an application on Clever Cloud](/clever-cloud-overview
 ## Necessary information
 
 * The application must be located at the **root** of the git repository.
-* Play 2.4 need **Java 8** to work. You need to [select Java 8](https://www.clever-cloud.com/doc/java/select-java-version/)
-for your application.
+* Starting from **Play 2.4**, your application needs **Java 8** to run. Please, read [select java version](https://www.clever-cloud.com/doc/java/select-java-version/) for more information.
```
Comment (divarvel) [inline runtimes/java/play-framework-2.md]: Please read

File: runtimes/scala/play-framework-2.md [hunk 6]
```diff
@@ -20,8 +20,7 @@ Refer to the page [Deploy an application on Clever Cloud](/clever-cloud-overview
 ## Necessary information
 
 * The application must be located at the **root** of the git repository.
-* Play 2.4 need **Java 8** to work. You need to [select Java 8](https://www.clever-cloud.com/doc/java/select-java-version/)
-for your application.
+* Starting from **Play 2.4**, your application needs **Java 8** to run. Please, read [select java version](https://www.clever-cloud.com/doc/java/select-java-version/) for more information.
```
Comment (divarvel) [inline runtimes/scala/play-framework-2.md]: Please read

**Pull request merged**